Appsody_Cert_Criteria.md

```diff
@@ -0,0 +1,83 @@
+# Certification criteria for Appsody application stacks
+
+This document describes the criteria an Appsody application stack must implement to be considered a certified application stack.
+
+## Certification Criteria
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in [BCP
+14](https://tools.ietf.org/html/bcp14)
+[[RFC2119](https://tools.ietf.org/html/rfc2119)]
+[[RFC8174](https://tools.ietf.org/html/rfc8174)] when, and only when, they
+appear in all capitals, as shown here.
+
+### Stack structure
+
+* A certified Application Stack MUST also be an [Application Stack](https://github.com/IBM/cp4apps-hub/tree/devfile_cert#application-stacks), having the necessary files needed to fulfill core scenarios by the architect (Champ) and the developer (Jane).  
```

Comment:
should the app stack link reference Appsody Stacks, since this is the Appsody readme?

Reply:
The original link was to the Appsody documentation but I felt having a generic high level description would be better unless we have an equivalent reference for odo.